docs/mapping.md

```diff
@@ -0,0 +1,286 @@
+# Mapping configuration
+
+## Introduction
+
+A mapping configuration is made of 3 objects: 
+- an IIDM network
+- a time series store (in csv format)
+- a mapping groovy script using the [mapping DSL](#mapping-dsl) described below
+
+The mapping process is mainly used through Metrix simulator but it may be executed separately in order to check its intermediate results.
+
+With only required options, the mapping does not produce any output. In adding option `--mapping-synthesis-dir`, it will output a [synthesis](#synthesis) of the mapping, allowing you to check the consistency of the produced multi cases. If you want to go deeper in the mapping process, the option `--check-equipment-time-series` will produce the full time series mapping. Lastly, the option `--network-output-dir` will output the network case with mapped values for the steps requested.
+
+## Mapping DSL
+
+The purpose of the mapping is to map time series values on network elements. As it is a groovy script, it can create composite time series or modify the network.
+
+### General parameters
+
+The general parameters are used to configure globally the mapping.
+
+```groovy
+parameters {
+    toleranceThreshold // Tolerance threshold when comparing the mapped power to the limit defined by the network element (default : 0.0001) 
+}
+``` 
+
+### Time series mapping to network
+
+There is one mapping function for each type of network elements:
+
+```groovy
+mapToGenerators {...}
+mapToLoads {...}
+mapToHvdcLines {...}
+mapToBoundaryLines {...}
+mapToBreakers {...}
+mapToTransformers {...}
+mapToPhaseTapChangers {...}
+mapToRatioTapChangers {...}
+mapToLccConverterStations {...}
+mapToVscConverterStations {...}
+```
+
+the general syntax is: 
+
+```groovy
+mapToXXX {
+ variable variableName //optional, the name of the element variable to map the time series with. Each element have a default mapped variable (defined below)
+ timeSeriesName 'timeseriesName' // name of the time series to map
+ filter {...} // a filter predicate allowing to select which item of the element type we wish to map
+ distributionKey {...} // optional, a repartition key that will define how the time serie values will be distributed for each selected items (default is equipartition : val / N)
+}   
+```
+The element of the general syntax are described below. 
+
+Note that for the `mapToBreakers` function, the `variable` and `distributionKey` are always ignored.
+
+#### Time series
+
+With the `timeSeriesName` variable you can map any time series by referring to its name. The available time series must exist in the input data set or be created within the groovy script. For more details about time series management, refer to the [time series description](../../data/timeseries.md).
```

Comment:
The link towards timeseries is not working yet because the corresponding page is not up yet (you kept the old link from the website if I am not mistaken). Could you add a TODO here so that it does not look like a genuine path?